src/EnergyPlus/DaylightingManager.cc

```diff
@@ -10413,13 +10436,6 @@ void MapShadeDeploymentOrderToLoopNumber(EnergyPlusData &state, int const enclNu
                                              "MapShadeDeploymentOrderToLoopNumber: found unassociated window for enclosure " + thisSolEnclosureName);
```

Comment:
It would be helpful to include the surface name in this warning. This check seems inefficient. I know it only happens during inits, but this is looping through all windows in the enclosure for every window that has a shading control. But I see how it's a crosscheck that the shaded surface is actually an exterior window for this enclosure. So, ok. But adding the surface name here would be very useful.